packages/components/src/internal/components/entities/utils.test.ts

```diff
@@ -462,3 +463,13 @@ describe('get cell key helpers', () => {
         });
     });
 });
+
+describe('createEntityParentKey', () => {
+    test('without id', () => {
+        expect(createEntityParentKey(new SchemaQuery('schema', 'query'))).toBe('schema:query');
+    });
+    test('with id', () => {
+        expect(createEntityParentKey(new SchemaQuery('schema', 'query'), 'id')).toBe('schema:query:id');
```

Comment:
Shouldn't these be expecting the `|` as the divider?

Reply:
updated